zone-service/src/main/java/org/cloudfoundry/identity/uaa/zone/OrchestratorZoneService.java

```diff
@@ -221,11 +222,20 @@ public void createZone(OrchestratorZoneRequest zoneRequest) throws OrchestratorZ
         }
     }
 
+    private String getName(OrchestratorZoneRequest zoneRequest) throws OrchestratorZoneServiceException {
```

Comment:
Purpose of this PR and corresponding user story is to fix format of the error message to valid json.  Adding name validation is not part of this story.  Also, this change is not covered by any tests. These changes should be done in the context of input validation story.

Remove changes to OrchestratorZoneService class.

Reply:
Consider refactoring OrchestratorZoneService deleteZone() method.  To be consistent with the rest of the public methods, it should not return ResponseEntity<?>; it should return void instead.  Error handling and ResponseEntity should be moved into OrchestratorZoneController.

In OrchestratorZoneService refactor deleteZone() :
```
    public void deleteZone(String zoneName) {
        IdentityZone previous = IdentityZoneHolder.get();
        try {
            ...
            if (publisher != null && zone != null) {
                ...
                return;
            } else {
                throw new Exception(...);
            }
        } finally {
            IdentityZoneHolder.set(previous);
        }
    }
```

In OrchestratorZoneController, refactor deleteZone():
```
    @DeleteMapping
    public ResponseEntity<?> deleteZone(@NotBlank(message = MANDATORY_VALIDATION_MESSAGE) @RequestParam String name) {
        return ResponseEntity.accepted().build();
    }
```
That refactoring fits under "fix error response handling..."

Reply:
Done. Thanks!

Reply:
> Purpose of this PR and corresponding user story is to fix format of the error message to valid json. Adding name validation is not part of this story. Also, this change is not covered by any tests. These changes should be done in the context of input validation story.
> 
> Remove changes to OrchestratorZoneService class.

@babamishra9 : Done. Thanks!